csv.md

```diff
@@ -0,0 +1,124 @@
+- All columns should have values for every observation, except for the observation, observation status, or attribute type columns.
+
+## Column
+
+### Headers
+
+Column headers should be in lowercase and snake case (e.g. `column_header`). This is to ensure consistency and readability. Column headers should also be unique, and should not contain any special characters (e.g. `!@#$%^&*()`). This even includes the pound sign (i.e. `£`), which should be replaced with `gbp` when appropriate.
+
+Related columns should have the same prefix (e.g. `geo_code`, `geo_name`, `geo_type` or `millions_gbp` and `millions_gbp_status`, or even `observation` and `observation_status`). This is to ensure that related columns are grouped together when sorted alphabetically, and to make it easier to find concepts whose values are spread across multiple columns.
+
+### Types
+
+There are five types of columns, and each CSV file should contain at least two of them.
+
+#### Observation
+
+Observation columns should only contain numbers. Suppressed or missing values should be left blank. If a value is suppressed, there should be a related column explaining the suppressed value. This is referred to as an observation status column (i.e. a special kind of attribute column called "observation status"). When dealing with whole numbers (i.e. counts of people) and where there isn't scaling (i.e. thousands, millions, etc.), the number should be expressed as an integer. When dealing with decimal numbers (i.e. percentages, indexes, scaled currency counts), the number should be expressed as a decimal number.
+
+**Note:** Try and keep the same number of decimal places for all values in a given column. This will make it easier to read, and not imply false precision.
+
+#### Dimension
+
+Dimension columns (otherwise known as factors) are used to identify the observation through a combination of concepts. Where each dimension in a CSV is filtered to a specific value there should only be one observation. In relational databases terminology all dimensions form to be a unique composite key. Some examples of dimensions are:
+
+- `government_year` with one value being `2019-2020` (i.e. the period of `April 2019 to March 2020`)
+- `geography_code` with one value being `E09000001` (i.e. the nation of `England`)
+- `sic_2007` with one value being `01.11` (i.e. the concept `Growing of cereals (except rice), leguminous crops and oil seeds`)
+
+To improve human readability and to ensure machine readability, ensure that codes in separate columns from the related human readable labels or descriptions. While the labels and additional information can be useful for humans, mixing them in the same column with codes makes users have to clean the data. It is also important to ensure that related columns are grouped together when sorted alphabetically, and to make it easier to find the human readable values.
```

Comment:
Edit the line to 'ensure that codes **are** in separate columns'. I think the sentence makes more sense that way.